lib/ansiblelint/utils.py

```diff
@@ -435,14 +435,16 @@ def _sanitize_task(task: dict) -> dict:
 
 # FIXME: drop noqa once this function is made simpler
 # Ref: https://github.com/ansible-community/ansible-lint/issues/744
-def normalize_task_v2(task: dict) -> dict:  # noqa: C901
+def normalize_task_v2(task: dict, options=options) -> dict:  # noqa: C901
```

Comment:
Using mutable objects as defaults is a known and dangerous antipattern that can lead to hard to follow behaviors.